asserts/database.go

```diff
@@ -311,9 +338,12 @@ func (db *Database) FindMany(assertionType *AssertionType, headers map[string]st
 	foundCb := func(assert Assertion) {
 		res = append(res, assert)
 	}
-	err = db.bs.Search(assertionType, headers, foundCb)
-	if err != nil {
-		return nil, err
+
+	for _, bs := range db.backstores {
+		err = bs.Search(assertionType, headers, foundCb)
```

Comment:
discussed this with @emgee, the check in Add is there to prevent the issue, but yes there are probably ways outside of the library control to get there, outside of tampering probably involving serious confusion on our side in terms of key mgmt. That's more related to the general work of having some kind of global sanity checker (one of my next todos)...  coalescing things here would mostly paper over and add complexity so we agreed to leave as is